tests/unit/data_encoding/test_metadata_encoding_decoding.py

```diff
@@ -0,0 +1,33 @@
+"""
+To Test the module metadata_encoding_decoding.py
+"""
+
+from modules import data_encoding
+
+
+def test_encoding_metadata() -> None:
+    """
+    Function to test encoding
+    """
+
+    # Step 1: Create a worker_name and PositionGlobal object
+    worker_name = "communications_worker"  # =3 in Worker_Enum
+    number_of_messages = 5
+
+    # Step 2: Encode the WorkerEnum ID and number of messages
+
+    encoded_bytes = data_encoding.metadata_encoding_decoding.encode_metadata(
```

Comment:
Same for this file, use tuple unpacking: `result, encoded_bytes = ...` and same for decoding